src/server.rs

```diff
@@ -502,6 +551,7 @@ impl ServerState {
             RequestAction::None => {
                 return ResponseCode::BadRequest( Some("Invalid request".to_owned()) );
             },
+            RequestAction::TestSequenceNumber(b) => { return ResponseCode::OldPacket; },
```

Comment:
Why does it always return the same thing? Should it be doing something with `b`?

Reply:
This was my initial dive into seeing the behavior of a real case of spamming a random sequence number to the server. That's why it looks rather hacked together (because it was lol) which is why nothing was done with 'b'. Never meant to commit it but it accidentally slipped through one night. It's only for bench testing and will be removed before any the final parts of reliability are merged into master.